.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 .env
+yarn.lock
+package-lock.json
```

Comment:
package-lock.json should remain in source control

Reply:
ok, i did it because it gives a warning when using yarn